core/src/main/java/org/infinispan/util/Closeables.java

```diff
@@ -0,0 +1,38 @@
+package org.infinispan.util;
+
+import java.util.Iterator;
+
+import org.infinispan.commons.util.CloseableIterator;
+import org.reactivestreams.Publisher;
+
+import io.reactivex.Flowable;
+import io.reactivex.disposables.Disposable;
+
+/**
+ * @author wburns
+ * @since 9.0
```

Comment:
Bah, another Closeables class... btw. the since tag does not match.

Reply:
Yeah unfortunately reactive streams isn't in commons yet :(

My eclipse isn't updated - thanks 👍

Reply:
Would it hurt to rather add the dependency there? We want to eventually keep up parity with remote clients anyway...

Reply:
Afaik it shouldn't hurt. I agree, I was trying to delay the introduction of it until we absolutely needed it in the client.